lib/redis_memo/memoizable/invalidation.rb

```diff
@@ -143,15 +126,18 @@ def bump_version_later(key, version, previous_version: nil)
       end
     end
 
+    @@bump_version_sha = nil
     def bump_version(task)
       RedisMemo::Tracer.trace('redis_memo.memoizable.bump_version', task.key) do
         ttl = RedisMemo::DefaultOptions.expires_in
         ttl = (ttl * 1000.0).to_i if ttl
-        RedisMemo::Cache.redis.eval(
+        RedisMemo::Cache.redis.run_script(
           LUA_BUMP_VERSION,
+          @@bump_version_sha,
           keys: [task.key],
           argv: [task.previous_version, task.version, RedisMemo::Util.uuid, ttl],
         )
+        @@bump_version_sha ||= Digest::SHA1.hexdigest(LUA_BUMP_VERSION)
```

Comment:
This should happen before L134 right?

Reply:
I put this after as a very minor optimization - the first time this script is called, the sha will be nil since we don't expect the script to be loaded yet. If the sha is nil, we don't call evalsha

Reply:
Though maybe for code clarity, and also because this script probably will not change much at all, it's not worth it 😅

Reply:
Moved it to before calling `run_script`

Reply:
> Moved it to before calling run_script

Yeah that seems to be better. Also, the first time we calculate the SHA1, the script might have already be loaded on Redis (from another process earlier), so we can avoid the first eval most of the time (until we introduced some changes to the Lua script or the switched to a different Redis server)